scipy/stats/tests/test_stats.py

```diff
@@ -4042,7 +4042,7 @@ class Test_ttest_ind_permutations():
         (a, b, {'random_state': 0, "axis": 1}, p_d),
         (a, b, {'random_state': np.random.RandomState(0), "axis": 1}, p_d),
         (a2, b2, {'equal_var': True}, 0),  # equal variances
-        (rvs1, rvs2, {'axis': 0, 'random_state': 0}, p_d_big),  # bigger test
+        (rvs1, rvs2, {'axis': -1, 'random_state': 0}, p_d_big),  # bigger test
```

Comment:
These changes are needed due to [this](https://github.com/scipy/scipy/pull/13312/files#r655710752). Should be able to use `axis=1` here, too, but this would require at least one other tweak to the permutation t-test.